terra/asset.json

```
@@ -2749,6 +2749,7 @@
     "symbol": "NICO",
     "decimals": "18",
     "type": "cw20"
```

```suggestion
    "type": "cw20",
```

Please help to add the missing comma